go/pkg/sysdb/coordinator/table_catalog.go

```diff
@@ -540,13 +540,16 @@ func (tc *Catalog) softDeleteCollection(ctx context.Context, deleteCollection *m
 		// Generate new name with timestamp and random number
 		oldName := *collections[0].Collection.Name
 		newName := fmt.Sprintf("_deleted_%s_%s", oldName, *types.FromUniqueID(deleteCollection.ID))
+		compactionTime := time.Now()
 
 		dbCollection := &dbmodel.Collection{
-			ID:        deleteCollection.ID.String(),
-			Name:      &newName,
-			IsDeleted: true,
-			Ts:        deleteCollection.Ts,
-			UpdatedAt: time.Now(),
+			ID:                      deleteCollection.ID.String(),
+			Name:                    &newName,
+			IsDeleted:               true,
+			Ts:                      deleteCollection.Ts,
+			UpdatedAt:               compactionTime,
+			SizeBytesPostCompaction: 0,
+			LastCompactionTimeSecs:  uint64(compactionTime.Unix()),
```

Comment:
Do we want to not update the last compaction time but instead add a deleted_at field? I can imagine in debugging cases that this distinction would be valuable. 

Would love @rohitcpbot's opinion

Reply:
I think we need `LastCompactionTimeSecs` to be set here in order for the calculation to be correct, right?